src/sokoban.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef __SOKOBAN_HPP__
+#define __SOKOBAN_HPP__
+#include <vector>
+#include <string>
+#include "elements.hpp"
+
+class Sokoban {
+    /* Sokoban Attribute */
+    protected:
+        std::vector<std::string> current_board;
+        std::string movesSequence;
+        unsigned int current_level;
+        unsigned int moves_count;
+
+    /* Sokoban Methods */
+    public:
+        Sokoban();
+        Sokoban(unsigned int inputNumber);
```

Comment:
```suggestion
        Sokoban(unsigned int level);
```
As mentioned, this will need to be a vector of levels or some other dynamic representation.